.circleci/config.yml

```diff
@@ -185,6 +300,12 @@ jobs:
             --build-arg COMMIT_ID=${CIRCLE_SHA1}
             --build-arg APP_BRANCH=${CIRCLE_BRANCH}
 
+          # ========================================================================== #
```

Comment:
I would be really tempted to take out an the python stuff and just keep it as legacy rather than trying to run both together. We are either going to run with one or the other not both side by side.

Reply:
So long as there's python code in the repo I'd prefer to have tests for the python code in CI. However, as soon as we're live with the Ruby code I'd advocate for taking both the python CI and the python code out. At that point it belongs in the git history, not git HEAD.

Reply:
Ok, I chatted this through with David and am happy for this to be the way forward.